src/admin/menu/views/MenuDetail.scss

```
@@ -29,3 +35,26 @@
 .c-menu-detail__nav-label {
 	margin: 0 auto 0 ($g-spacer-unit * 3);
 }
+
+.c-admin__header {
+	border-bottom: 0.1rem solid #d6dee3;
+	background-color: #edeff2;
+	display: flex;
+	padding: 20px 40px;
+
+	.c-admin__page-title {
+		font-size: 2.4rem;
+		font-family: 'Avenir LT Std', 'Avenir Next', Avenir, Montserrat, sans-serif;
+		font-weight: 500;
+	}
+}
+
+.c-admin__content {
+	margin: 40px;
```

always use rem units. They usually are px value divided by 10. Rem units offer better support for people that have poor vision since they can set their browser to make rem units bigger and easier to read.
```suggestion
	margin: 4rem;
```